src/Host/Broker/Impl/Security/SecurityOptions.cs

```diff
@@ -14,5 +14,15 @@ public class SecurityOptions {
         /// Friendly name of the certificate installed for the TLS (SSL)
         /// </summary>
         public string X509CertificateName { get; set; }
+
+		/// <summary>
+		/// Path to the certificate for use with TLS (SSL)
+		/// </summary>
+		public string X509CertificateFile { get; set; }
+
+		/// <summary>
+		/// Password to access private key in the certificate <see cref="X509CertificateFile"/>
+		/// </summary>
+		public string X509CertificatePassword { get; set; }
```

Comment:
This will be plain string stored in the config json file. It is a passphrase that protects the private key in a PFX cert file. The issue with setting a pass-phrase with the PFX file is that each time the server restarts some one will have to enter this pass-phrase. So storing it in the config file is one way to make this easy. The other option is to use a PFX file without a pass-phrase. The security here boils down to the integrity of the system itself. If a hacker can get access to the config file, the system is already compromised. 

Now, as for this property. I could not find a way to get configuration binding to bind to a SecureString. If there is a way let me know and I can make the change.